src/Model/Table/FilesTable.php

```diff
@@ -29,7 +29,10 @@ public function initialize(array $config)
         $this->setDisplayField('id');
         $this->setPrimaryKey('id');
 
-        $this->addBehavior('Josegonzalez/Upload.Upload', ['file']);
+        $this->addBehavior('Josegonzalez/Upload.Upload', ['file' => [
+            'path' => 'webroot{DS}files{DS}{field-value:event_id}{DS}',
+            'restoreValueOnFailure' => false
```

Comment:
What does this do?

Reply:
This has to do with if a value is restored on the model if a DB update fails, something we weren't using anyway. however, it has an underlying effect of removing the path when trying to update during a class copy. So, away it goes